samples/Nancy.Demo.Authentication.Stateless/SecureModule.cs

```diff
@@ -1,6 +1,7 @@
 namespace Nancy.Demo.Authentication.Stateless
 {
-    using Nancy.Demo.Authentication.Stateless.Models;
+  using System;

```

Comment:
Brill. Can you squash those 2 commits into 1. And the ill merge it :)

Thanks

On Thursday, 25 February 2016, Andrei Shirobokov <notifications@github.com>
wrote:

> In samples/Nancy.Demo.Authentication.Stateless/SecureModule.cs
> <https://github.com/NancyFx/Nancy/pull/2312#discussion_r54134505>:
>
> > @@ -1,6 +1,7 @@
> >  namespace Nancy.Demo.Authentication.Stateless
> >  {
> > -    using Nancy.Demo.Authentication.Stateless.Models;
> > +  using System;
>
> Add one more commit. Hope now is all right :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2312/files#r54134505>.
>